src/indexers/indexer.utils.ts

```diff
@@ -45,7 +45,7 @@ export async function vaultToSnapshot(chain: Chain, vaultDefinition: VaultDefini
     address: vaultDefinition.vaultToken,
     requireRegistry: false,
     state: VaultState.Open,
-    version: VaultVersion.v1,
+    version: vaultDefinition.version ?? VaultVersion.v1,
```

Comment:
nullish collision is for rendering mostly, do u need empty string in case?

Reply:
what do you suggest then like `vaultDefinition.version || VaultVersion.v1`?